Plugin.cs

```diff
@@ -33,30 +32,39 @@ public class Plugin
         public static string noArrowsCharacteristicName = "NoArrows";
 
         [Init]
-        public void Init(IPALogger pluginLogger)
+        public void Init([Config.Name(nameof(SongCore) + "/" + nameof(SongCore))]Config conf, IPALogger pluginLogger)
```

Comment:
This apparently breaks the game when the SongCore sub folder isn't present.

Reply:
Still not solved with the latest BSIPA.

Reply:
Apparently pushed the changes onto a branch of my SongCore fork and not on this branch, changes have been included now.

Reply:
After some back-and-forth, it is now truly fixed 🙂